### ISSUE TITLE
Reader - discover - does not preserve focus when selecting tabs for a11y.

### ISSUE THREAD

**Issue by Addison-Stavlo** [Automattic/wp-calypso#101244]
When navigating through the discover page with keyboard, when selecting a tab, the focus restarts at the top of the page:

![Image](https://github.com/user-attachments/assets/e4ab799d-b9fd-41d5-8770-59719f8456e2)

This isn't the best experience wrt a11y, but also not blocking.

I would expect to be able to toggle between these more easily similar to the tags page:

![Image](https://github.com/user-attachments/assets/0a430f7c-03d1-4be0-bdf2-19cbb0ec8304)

**Comment by davemart-in** [comment]
Related: https://github.com/Automattic/wp-calypso/issues/101245

**Comment by arunsathiya** [comment]
📌 FINDINGS/SCREENSHOTS/VIDEO<br>I can reproduce the issue.<br><br>📌 ACTIONS<br>- Triaged; moving to Loop team on Linear as this issue seems to be on their radar already.

**Comment by arunsathiya** [comment]
Update to my last note - I haven't updated the team on Linear, because I realized this issue may be intentionally remaining on the Calypso GitHub because the work touches Calypso. I'll leave things as they are now.

**Comment by Addison-Stavlo** [comment]
> I haven't updated the team on Linear, because I realized this issue may be intentionally remaining on the Calypso GitHub because the work touches Calypso

🤔 I thought these were supposed to get migrated to and sync'd with our linear, I see the label was added but this one doesn't seem to be over there yet. @davemart-in may have more info

**Comment by davemart-in** [comment]
There has been technical difficulties with wp-calypso issues. Here's the latest that I'm aware of: p1745849996826129/1745512838.421109-slack-C08CTE09M7H
